### ISSUE TITLE
Datamodule not destroyed

### ISSUE THREAD

**Issue by leonardorame** [silvioprog/brookframework#116]
Hi Silvio!, this looks like a bug or I'm doing something in the wrong way.

My .lpr creates a TDataModule this way:

```
begin
  BrookApp.CreateForm(TDataModule1, DataModule1);
  BrookApp.Run;
end.    
```

When the app finishes (this is a CGI), the DataModule1 is not destroyed. I've found this by taking a look at the PostgreSql logs on my server, and found A LOT of "could not receive data from client: Connection reset by peer." messages.

So tracing the communication I found the DataModule's create event connects to the database properly, but onDestroy is never called, so a quick'n dirty solution is this:

```
begin
  BrookApp.CreateForm(TDataModule1, DataModule1);
  BrookApp.Run;
  DataModule1.Free; // with this the errors dissapear.
end.    
```

**Comment by silvioprog** [comment]
I've been busy these days. =( But I'll test it soon.

**Comment by silvioprog** [comment]
Fixed in 36370eb01a39b3825a753f93179afc9c6ae596c7. Thanks for reporting!
